lib/error/lib/merr.c

```diff
@@ -107,18 +109,23 @@ merr_strerror(merr_t err, char *buf, size_t buf_sz)
 }
 
 char *
-merr_strinfo(merr_t err, char *buf, size_t buf_sz, size_t *need_sz)
+merr_strinfo(
+    const merr_t err,
+    char *const buf,
+    const size_t buf_sz,
+    merr_stringify stringify_ctx,
+    size_t *const need_sz)
 {
     int ret = 0;
     size_t sz = 0;
     const char *file = NULL;
+    const int16_t ctx = merr_ctx(err);
```

Comment:
Why not unsigned?

Reply:
Figured error codes ala CURLCode were more likely to be signed rather than unsigned.

Reply:
Ah, that's why I'm confused.  It's not an error context, it's just an error code from some other subsystem...